src/RegExLib.Core/Entities/Expression.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using RegExLib.Core.Events;
+using RegExLib.Core.Interfaces;
+using RegExLib.SharedKernel;
+
+namespace RegExLib.Core.Entities
+{
+    public class Expression : BaseEntity
+    {
+        public int AuthorId { get; private set; }
+        public string Title { get; }
+        public string Pattern { get; }
+        public string Description { get; }
+        public Author Author { get; private set; }
```

Comment:
If we're following DDD and using Aggregates it seems like Author has to be the aggregate root since it has a collection of Expressions. Assuming that's the current path we're on, we want navigation properties to be one way, so the Author property should be removed here. If you have an Expression and you need its Author, you probably already have it (since it's the root), but if not, you can always get it since you have the AuthorId property.